tests/security/authz_tcp/authz_tcp_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package authztcp
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+
+	"istio.io/istio.io/pkg/test/istioio"
+)
+
+//https://istio.io/docs/tasks/security/authorization/authz-tcp/
+//https://github.com/istio/istio.io/blob/release-1.5/content/en/docs/tasks/security/authorization/authz-tcp/index.md
+func TestAuthzTCP(t *testing.T) {
+	framework.
```

Comment:
@justinpettit I think it would be a good idea to add the following line to disable the test and then we should merge this PR. Otherwise, you're going to need to regen the snip file whenever the generated output is tweaked (for example, when #7233 merges). We can get it working in a followup PR.

```suggestion
	t.Skip("Not working yet")
	framework.
```

Reply:
Thanks, Frank.  We're doing a bit more research on the underlying issue.  Let me see how that works out.  If there doesn't seem to be an underlying issue that needs to be addressed, I'll do resubmit with your suggestion.